pack.yaml

```diff
@@ -8,7 +8,7 @@ keywords:
    - ncm
    - npm
    - monitoring
-version: 0.7.11
+version: 0.7.12
```

Comment:
Is this not a new feature and should be a minor version bump and not a patch according to Semver?